sentry2csv/sentry2csv.py

```diff
@@ -176,7 +169,12 @@ def main():
     parser.add_argument("--enrich", help="Optional mappings of event metadata")
     parser.add_argument("--token", metavar="API_TOKEN", nargs=1, required=True, help="The Sentry API token")
     parser.add_argument(
-        "--host", metavar="HOST", nargs=1, required=False, help=f"The Sentry host [default: {SENTRY_HOST}]"
+        "--host",
+        metavar="HOST",
+        nargs=1,
+        required=False,
+        default=SENTRY_HOST,
```

Comment:
I believe this needs to be a list (i.e., `default=[SENTRY_HOST]`)

Reply:
Thank you!